src/server/routing/router-development.ts

```diff
@@ -58,20 +83,27 @@ authRouterDevelopment.get(
         const queryString = queryEntries.length
           ? `?${queryEntries.map(([key, val]) => `${key}=${val}`).join('&')}`
           : '';
-        return `<li><a href=${BffEndpoints.AUTH_LOGIN_DIGID}/${userName}${queryString}>${userName}</a>`;
+        return `<li><a href=${authRoutes.AUTH_LOGIN_DIGID}/${userName}${queryString}>${userName}</a>`;
```

Comment:
Het auth gedeelte zou nu van `AUTH_LOGIN_DIGID` afgehakt kunnen worden. En ook bij andere relevante paden

Reply:
Zou kunnen maar is niet voor deze PR omdat het niet triviaal is. search.replace is niet voldoende.

Reply:
Ah jammer, dan maar zo laten